src/app/modules/item/pages/item-progress/itemProgressLabel.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Group } from '../../../group/http-services/get-group-by-id.service';
+import { ItemType } from '../../../../shared/helpers/item-type';
+
+@Pipe({ name: 'itemProgressLabel' })
+export class ItemProgressLabelPipe implements PipeTransform {
+  transform(type: ItemType, watchedGroup?: Group): any {
+    if (!!watchedGroup && [ 'Chapter', 'Task', 'Course' ].includes(type)) {
```

Comment:
As written somewhere else:
Ok. What I mean actually is that here, you are in a if/else which, depending that you watch another group or not, you have "your situation" or "situation of ... ". The item type does not change anything to that. (possibly the whole component will not be displayed to "skills" but that's not the point here.
So, in short, just base your condition (and pipe) only on watched group.

Reply:
Fixed